### DISCUSSION TITLE
Synced Lyrics again

### DISCUSSION THREAD

**Discussion by Hajuskin** [jcorporation/myMPD#448] [General]
I've just tested 6.12.0, nice to have the synced lyrics working :)
A few little things I want to mention:

- The big line spacing is in my opinion not ideal for following the scrolling text, a normal spacing might be better
- Empty lines in the lyrics are not displayed
- If you scroll the text manually, it will jump back when the next line is played.

Also, is it possible to have a soft scroll effect for the text?

**Comment by jcorporation** [comment]
#449
